### ISSUE TITLE
Data for hair reconstruction works not included?

### ISSUE THREAD

**Issue by kenshi84** [facebookresearch/multiface#19]
Hi, thanks a lot for sharing such a great dataset with the world.

The README says that this dataset was used in the two previous hair reconstruction works ("Strand-accurate Multi-view Hair Capture" and "Human Hair Inverse Rendering using Multi-View Photometric data"), but as far as I can see, none of the available 13 subjects data seem to contain the input images depicted in these two papers.

Is this simply a mistake of README and the mentions to these works should be removed?
Or can the relevant data be added in the future?

**Comment by cwuu** [comment]
Hi,
Only mugsy images are needed for these 2 papers. People can reproduce that paper using the public dataset, if they spend enough time to implement it. Thanks.

**Comment by kenshi84** [comment]
I appreciate your quick response, but I have to disagree.

These are taken from the [CVPR 2019 paper video](https://www.youtube.com/watch?v=RExALS8qIfo), showing 3 out of 6 of the real subjects demonstrated in the paper.
![image](https://user-images.githubusercontent.com/1090150/197100122-dd97cf14-2a6f-4ba3-af1e-96928ff2a41c.png)
![image](https://user-images.githubusercontent.com/1090150/197100220-06a91666-67de-4511-b107-1a457132e5a9.png)
![image](https://user-images.githubusercontent.com/1090150/197100242-98b7f772-7f9d-4b31-afec-0392bcd87fb8.png)

Here are the remaining 3 subjects where only 2 views are shown in the paper: 
![image](https://user-images.githubusercontent.com/1090150/197100584-787f211f-9adc-47f3-b558-08677f49f6c2.png)

None of the above appear to be identical to any of the subjects in your dataset:
![image](https://user-images.githubusercontent.com/1090150/197100887-abcd22c6-9c54-46b4-89f8-bd4a1a439035.png)

I suppose the input images for the CVPR 2019 paper were captured using the Mugsy v2 setup because the v1 setup concentrates most of the cameras to the frontal area, and there is a headrest occluding most of the hair in the back: (see red arrows) 
![image](https://user-images.githubusercontent.com/1090150/197101389-716973cd-7fbd-4d7d-ac02-52a101e37c9e.png)

**Comment by cwuu** [comment]
Hi,

Thanks for the follow-up questions, and I hope the following 3 clarifications will help: 

1) The dataset we used for the paper is from Mugsy v1. However, there's difference between the dataset we used for the paper and the dataset we released. We removed the headrest for these captures, and also asked the participants to rotate 180 deg., so that their heads are facing backwards. So, more cameras captured their back side of head, and less cameras captured their faces. 
2) People can still reproduce the results in the paper with the released dataset. The method reconstructs what is visible in the images, and it's just that the visible parts are different. 
3) Currently, we don't have further plan on releasing more images with back side of head. 

Let me know if you have any further questions.
